spec/syntax.md

```diff
@@ -275,6 +275,11 @@ SCSS:
   adjacent `/`s in a [`SlashListExpression`] may have no whitespace between
   them, so `//` is parsed as two slash separators in a slash-separated list.
 
+* A `ParentSelector` may appear anywhere in a `CompoundSelector`, rather than
+  just as the first `SimpleSelector`.
```

Comment:
This sentence is difficult for me to read, I think it makes sense if "as" is removed, is that what is intended?
```suggestion
  just the first `SimpleSelector`.
```

Reply:
This is intentional. Without "as", it reads as though `ParentSelector` may not appear _in_ the first `SimpleSelector`, when in fact it would _be_ the first `SimpleSelector`.